low-level/files/opt/seagate/sspl/bin/consul_config.json

```diff
@@ -1,9 +1,9 @@
 {
-    "watches": [
-          {
-            "type": "key",
-            "key": "sspl/config/SYSTEM_INFORMATION/log_level",
-            "args": ["/opt/seagate/cortx/sspl/bin/consume_cfg_change_alert"]
-          }
-    ]
+  "watches": [
```

Comment:
i dont see the need to touch this file. Make sure to maintain required indentation

Reply:
Done